scripts/analysis.py

```diff
@@ -105,13 +105,13 @@ def get_intercept(x, y, m, b):
 print(f"Y Velocity Random Walk: {accel_wn_intercept_y: .5f} m/s/sqrt(s) {accel_wn_intercept_y*60: .5f} m/s/sqrt(hr)")
 print(f"Z Velocity Random Walk: {accel_wn_intercept_z: .5f} m/s/sqrt(s) {accel_wn_intercept_z*60: .5f} m/s/sqrt(hr)")
 
-print(f"X Bias Instability: {accel_min_x: .5f} m/s^2 {accel_min_x*60*60: .5f} m/hr^2")
```

Comment:
Thanks, good spotting

Reply:
Could you actually change the line to be `accel_min_x*3600*3600` I want the units displayed in m/hr^2 because some IMU datasheets use these units.

Reply:
Okay, done! Some day the mix of units between datasheets, standards and software is going to drive me crazy. BTW, ome manufacturers give the units of accelerometer bias instability in mg, which is not the SI unit "milligrams", but [milli-g-force](https://en.wikipedia.org/wiki/G-force)...